resources/public/api/conf/1.0/schemas/gmr-types-schema.json

```diff
@@ -0,0 +1,513 @@
+        }
+      }
+    },
+    "ataDeclarations": {
+      "$id": "#/definitions/ataDeclarations",
+      "type": "array",
+      "description": "Details of one or more ATA Carnet declarations.  Must be provided prior to finalisation for ATA movements.",
+      "items": {
+        "$id": "#/definitions/ataDeclarations/items",
+        "type": "object",
+        "required": [
+          "ataCarnetId"
+        ],
+        "properties": {
+          "ataCarnetId": { "$ref": "#/definitions/ataCarnetId" },
+          "sAndSMasterRefNum": { "$ref": "#/definitions/sAndSMasterRefNum" }
+        }
+      }
+    },
+    "traderEORI": {
+      "$id": "#/definitions/traderEORI",
+      "type": "string",
+      "description": "The trader's GB EORI",
+      "examples": [
+        "GB123456789012"
+      ],
+      "pattern": "^GB[0-9]{12,15}$"
+    },
+    "palletReference": {
+      "$id": "#/definitions/palletReference",
+      "type": "string",
+      "description": "The reference number for the pallet holding the EIDR goods",
+      "examples": [
+        "ABSUE98287ABSUIJHD"
+      ],
+      "pattern": "^[A-Z0-9-]{1,200}$"
+    },
+    "emptyVehicle": {
+      "$id": "#/definitions/emptyVehicle",
+      "type": "object",
+      "description": "Declares that this is an empty movement and contains details about the empty movement.  Must be provided prior to finalisation for empty movements.",
+      "required": [
+        "isOwnVehicle"
+      ],
+      "properties": {
+        "isOwnVehicle": { "$ref": "#/definitions/isOwnVehicle" },
+        "sAndSMasterRefNum": { "$ref": "#/definitions/sAndSMasterRefNum" }
+      }
+    },
+    "customsDeclarations": {
+      "$id": "#/definitions/customsDeclarations",
+      "type": "array",
+      "description": "Details of the customs declarations within this consignment",
+      "items": {
+        "$id": "#/definitions/customsDeclarations/items",
+        "type": "object",
+        "required": [
+          "customsDeclarationId"
+        ],
+        "properties": {
+          "customsDeclarationId": { "$ref": "#/definitions/customsDeclarationId" },
+          "sAndSMasterRefNum": { "$ref": "#/definitions/sAndSMasterRefNum" }
+        }
+      }
+    },
+    "sAndSDeclarations": {
+      "$id": "#/definitions/sAndSDeclarations",
+      "type": "array",
+      "description": "Details of the safety and security (S&S) declarations within this consignment",
+      "items": {
+        "$id": "#/definitions/sAndSDeclarations/items",
+        "type": "object",
+        "required": [
+          "masterRefNum"
+        ],
+        "properties": {
+          "masterRefNum": { "$ref": "#/definitions/sAndSMasterRefNum" }
+        }
+      }
+    },
+    "transitDeclarations": {
+      "$id": "#/definitions/transitDeclarations",
+      "type": "array",
+      "description": "Details of the transit declarations within this consignment",
+      "items": {
+        "$id": "#/definitions/transitDeclarations/items",

```

Comment:
I think a lot of the stuff in this file isn't needed for the port-api and was probably copied across when the service was first created. Would be good to cut this file down to the essentials before we go public.